### PR TITLE
Fix dependencies import.

### PR DESCRIPTION
**Pull request by keuller**
I've fixed import dependencies and I improve the README.md file.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
   "repository": "developit/babel-preset-preact",
   "main": "index.js",
   "dependencies": {
-    "babel-plugin-syntax-flow": "^6.0.2",
     "babel-plugin-syntax-jsx": "^6.0.2",
-    "babel-plugin-transform-flow-strip-types": "^6.0.2",
     "babel-plugin-transform-react-jsx": "^6.0.2"
+  },
+  "devDependencies": {
   }
```
Comment (jescalan) [inline package.json:15]: This can be removed entirely since it's empty

**Review by jescalan — COMMENTED** [review]
Would love to see this merged!

**Review by developit — APPROVED** [review]
LGTM!

**Comment by developit** [comment]
Published as 1.1.0 :)

**Pull request merged**